content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+---
+title: "Deploy Edge Cluster"
+metaTitle: "Deploy Edge Cluster on VMWare VMs"
+metaDescription: "Learn how to use the Edge installer ISO image to prepare Edge hosts, create a cluster profile using the provider image, and deploy a cluster on those Edge hosts."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from "shared/components/common/PointOfInterest";
+
+# Deploy Edge Cluster
+Deploying a Kubernetes (K3s) cluster on Edge, also called Edge cluster, requires all Edge devices to be ready with the required dependencies, user data configurations, and an operating system (OS). 
+
+Maintaining consistency while preparing Edge devices at scale is always challenging for the IT/Site operations team. For example, imagine you are an IT administrator for a retail company that has decided to expand to 1000 new stores this year. The company wants to deploy K3s clusters on Edge for each new store. Assuming all Edge devices have the same configuration, your job is to prepare all Edge devices so the development team can deploy clusters on all those devices. 
+ 
+As an IT administrator, you will prepare a small set of Edge devices and deploy a sample cluster to check readiness for consistent deployment across all sites. After a successful sample deployment, here are the primary stages of deploying an Edge cluster in production:  
+
+- IT administrators build the Edge artifacts. 
+
+- Site operators prepare Edge devices with the installer ISO and identify the additional user data to install on those devices.
+
+- The development team prepares a cluster profile using the provider image and deploys the clusters.
+
+
+In this tutorial, similar to the primary stages outlined above, you will first build Edge artifacts (Edge installer ISO image and provider images) and use the installer ISO image to prepare Edge hosts. Next, you will use the provider image to create a cluster profile and then deploy a cluster on those Edge hosts. 
+
+
+Setting up Virtual Machines (VMs) as Edge hosts and deploying a cluster on those VMs is a less complex path to learning and gaining experience with Edge due to not having to connect to a physical Edge device. Therefore, this tutorial uses VMWare VMs as Edge hosts to test the installer ISO image's correctness and ease of use. The diagram below shows the main steps to prepare Edge hosts and deploy a cluster. 
+
+
+![An overarching diagram showing the tutorial workflow.](/tutorials/edge-native/clusters_edge_deploy-cluster_overarching.png)
```

Comment:
This diagram - cloning a template does not feel like the right word. Packer creates the template, and then GOVC is used to deploy an instance using the template as a blueprint.  So to me, step 5 is deploying VMs.
![CleanShot 2023-06-16 at 14 00 09](https://github.com/spectrocloud/librarium/assets/29551334/827b2e70-4840-4f3c-9da7-7a98fcb15a26)